packages/product-react/src/components/List/List.scss

```diff
@@ -0,0 +1,51 @@
+@import url('https://www-static.wework.com/apercu/apercu.css');
+@import url('https://www-static.wework.com/apercu/apercu_mono.css');
+
+$list-font-family: 'Apercu';
+$list-font-weight: normal;
+$list-block-display: block;
+$list-inline-block-display: inline-block;
+$list-vertical-align: middle;
+
+.ray-list-item {
+  font-family: $list-font-family;
+  font-weight: $list-font-weight;
+  border-radius: 4px;
+  background-color: rgb(247, 247, 247);
+}
+
+.ray-list-item__header {
```

Comment:
😂I'm sure I'm the culprit for these not being nested!! lol Sorry! but we can probably nest these like so:
```
.ray-list-item {
    &__header {
        ...
    }
}
```

Reply:
done